cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2004,3 +2004,74 @@ action.summary_index = 1
 action.summary_index._name = cyences
 action.summary_index.category = Authentication
 action.summary_index.alert_name = Authentication - Excessive Failed VPN Logins from a Source
+
+
+
+# ===============
+# Linux/Unix
+# ===============
+[Linux - Local User List - Lookup Gen]
+disabled = 1
+enableSched = 1
+alert.track = 0
+cron_schedule = 53 * * * *
```

Comment:
let's run it on 29th or 59th minute. (let's which one has less load)

Reply:
Is it to make it run before alert?

Reply:
There will not be a lot of data for these source types. So no need to worry about load.